Server/routes/users.js

```diff
@@ -6,13 +6,13 @@ let router = express.Router();
 
 
 router.post('/',(req,res) => {
-   const { errors, isValid} = validateInput(req.body);
-   if(isValid){
-     res.json({succes:true});
-   }
-   else{
-     res.status(400).json(errors);
-   }
+  const { errors, isValid} = validateInput(req.body);
+  if(isValid){
+    res.json({succes:true});
+  }
+  else{
```

Comment:
it would look better to put space after `if` and `else` and `{`. Will find the eslint rule that does this.